mindsdb/integrations/handlers/lightwood_handler/lightwood_handler/lightwood_handler.py

```diff
@@ -98,11 +98,17 @@ def predict(self, df, args=None):
             predictor = LightwoodHandler.get_predictor(predictor_path, predictor_code)
 
         dtype_dict = predictor.dtype_dict
+        embedding_mode = predictor.problem_definition.embedding_only or pred_args.get('return_embedding', False)
```

Comment:
does `predictor.problem_definition.embedding_only` exists in predictors trained with old versions of lightwood?should we add `hasattr` check here?

Reply:
Good catch, it is a new feature indeed.